ui/src/app/views/workspace-edit/component.ts

```diff
@@ -21,48 +21,54 @@ export enum WorkspaceEditMode { Create = 1, Edit = 2, Clone = 3 }
 export const ResearchPurposeItems = {
   methodsDevelopment: {
     shortDescription: 'Methods development/validation study',
-    longDescription: 'The primary purpose of the research is to develop and/or \
-    validate new methods/tools for analyzing or interpreting data (for example, \
-    developing more powerful methods to detect epistatic, gene-environment, or \
-    other types of complex interactions in genome-wide association studies). \
-    Data will be used for developing and/or validating new methods.'
+    longDescription: 'The primary purpose of the research is to develop and/or validate new \
+    methods/tools for analyzing or interpreting data (for example, developing more powerful \
+    methods to detect epistatic, gene-environment, or other types of complex interactions in \
+    genome-wide association studies). Data will be used for developing and/or validating new \
+    methods.'
   },
   diseaseFocusedResearch: {
     shortDescription: 'Disease focused research',
```

Comment:
The google doc specifies `Disease-focused research`

Reply:
The doc also calls for some additional text: `Select all that apply.` that I don't see on the page.

Reply:
I think that should be added as part of shifting to the ontology.

Reply:
I added the content detail to the ontology story, https://precisionmedicineinitiative.atlassian.net/browse/RW-1046